app/views/questionnaire.py

```diff
@@ -445,7 +445,7 @@ def _get_front_end_navigation(answer_store, current_location, metadata):
     completed_blocks = get_completed_blocks(current_user)
     navigation = Navigation(g.schema_json, answer_store, metadata, completed_blocks)
     block_json = SchemaHelper.get_block_for_location(g.schema_json, current_location)
-    if block_json is not None and (block_json['type'] == 'Questionnaire' or block_json['type'] == 'Interstitial'):
+    if block_json is not None and block_json['type'] in ('Questionnaire', 'Interstitial', 'Confirmation'):
```

Comment:
I'll leave this as-is for now, if that's okay. It's how it was previously and I don't think there's any harm in leaving it in there since block_json is assigned in the line just before and `get_block_for_location` has the potential to return `None`.